Description: Remove impl_bincode for now
 bincode 2 and unty aren't yet packaged in Debian.  This patch can be
 dropped once they are.
Author: Colin Watson <cjwatson@debian.org>
Forwarded: not-needed
Last-Update: 2025-08-20
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/

Index: b/Cargo.toml
===================================================================
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -48,10 +48,6 @@
 debugger_visualizer = []
 drain_filter = []
 drain_keep_rest = ["drain_filter"]
-impl_bincode = [
-    "bincode",
-    "unty",
-]
 may_dangle = []
 specialization = []
 union = []
@@ -79,11 +75,6 @@
 version = "1"
 optional = true
 
-[dependencies.bincode]
-version = "2"
-optional = true
-default-features = false
-
 [dependencies.malloc_size_of]
 version = "0.1"
 optional = true
@@ -94,11 +85,6 @@
 optional = true
 default-features = false
 
-[dependencies.unty]
-version = "0.0.4"
-optional = true
-default-features = false
-
 [dev-dependencies.bincode1]
 version = "1.0.1"
 package = "bincode"
